nics-tw/PETsARD
# test: create Issue550.ipynb

Evidence:
![image](https://github.com/nics-tw/PETsARD/assets/8596186/cf2304e8-1de4-4297-8706-4ec0423076bc)

---

- bug:
    - PETsARD/processor/base.py
        - add EncoderLabel to round() and astype(int) - 1c661a1cbf28b4e1db78394527a8831fbeea88b8 

- test:
    - demo/dev/Issue550.ipynb
        - create Issue550.ipynb - 20a8429daa515575e4b6cd180e270c88765ebe00 
        - test: upload validate testing - aa4c2a1a4435262d1a5e1cb06fe5dacb75e4aa7b 
        - delete Issue550.ipynb - 33fc6677c069bbd769a5428c7d043a3ee741d7c5 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207354312935967
